traits_test fails on msvc-14.1+ in 64bit debug mode with C1128 error

`b2 variant=debug address-model=64 toolset=msvc-14.1 libs/numeric/conversion/test//traits_test`
```
boost\libs\numeric\conversion\test\traits_test.cpp : fatal error C1128: number of sections exceeded object file format limit: compile with /bigobj
```

Either placing `<toolset>msvc:<cxxflags>/bigobj` or `<toolset>msvc:<debug-symbols>off` solves the problem.

--- a/test/Jamfile.v2
+++ b/test/Jamfile.v2
@@ -19,7 +19,7 @@ project numeric_conversion_unit_tests
 test-suite minimal 
     :
         [ run bounds_test.cpp ]
-        [ run traits_test.cpp ]
+        [ run traits_test.cpp : : : <toolset>msvc:<cxxflags>/bigobj ]
         [ run converter_test.cpp ]
         [ run udt_support_test.cpp ]
         [ run numeric_cast_test.cpp ]

